test/util/mathsEvaluator/test.js

```diff
@@ -148,14 +144,15 @@ define(['jquery', 'lodash', 'util/mathsEvaluator', 'json!test/util/mathsEvaluato
             try {
                 return evaluate(expr, variables).value;
             } catch (err) {
+                // eslint-disable-next-line
                 console.log(err);
                 return 'Syntax error!';
             }
         }
 
         function showResult(expression, result) {
-            var $expr = $('<p class="expression">' + expression + '</p>');
-            var $res = $('<p class="result">' + result + '</p>');
+            var $expr = $(`'<p class="expression">${expression}</p>'`);
```

Comment:
Despite the fact `$expr` is not the same (`'` are left in place after the refactoring) this test doesn't fail

Reply:
Actually, it applies to a visual test. The extraneous quotes were displayed on the example calculator, they are not part of any test. Not a big issue, but better to fix indeed, the result will look neater.